### DISCUSSION TITLE
Contribute to this project

### DISCUSSION THREAD

**Discussion by danny900714** [bytecodealliance/registry#272] [Q&A] [answered]
I'm interested in this project. How can I get the current development status and start contributing ?

**Comment by calvinrp** [comment, marked as answer]
@ZhaoTzuHsien Welcome! We would love to have you as a contributor. Our docs are very under developed, so it is difficult to get started without talking to someone.

We hold weekly meetings on Wednesdays at 1 PM - 2 PM CST, which is in a few hours from now. You can [join the Google Group]( https://groups.google.com/g/ba-sig-registries) to get the meeting invite. Let me know if you have any trouble joining.

The Bytecode Alliance Zulip chat is a good place to communicate and ask questions (in addition to this GitHub repo). [Zulip](https://bytecodealliance.zulipchat.com/#narrow/stream/352111-warg)

Also, I'm happy to schedule a call to intro the project and walk thru. Feel free to direct message me on the [BA Zulip](https://bytecodealliance.zulipchat.com/).
